modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -537,4 +556,88 @@ public List<UpdateEntry> get(Catalog catalog) {
             return bulkUpdateEntries;
         }
     }
+
+    private SystemView<?> createSystemViewsView() {
+        Iterable<SchemaAwareDescriptor<CatalogSystemViewDescriptor>> viewData = () -> {
+            int version = latestCatalogVersion();
+
+            Catalog catalog = catalog(version);
+
+            return catalog.schemas().stream()
```

Comment:
we supposed that we could have a few shcemas with a system views? Should we use concrete name of system schema instead of interesting by all of schemas?

Reply:
> we supposed that we could have a few shcemas with a system views?
from catalog's point of view, system view is just another object. like table or index. Thus, if anyone has managed to add system view to schema, it must be exposed by SYSTEM_VIEWS view